openslides/global_settings.py

```diff
@@ -11,7 +11,7 @@
 
 AUTHENTICATION_BACKENDS = ('openslides.users.auth.CustomizedModelBackend',)
 
-LOGIN_URL = '/login/'
+LOGIN_URL = '/users/'
```

Comment:
no. /users/login/ is a ajax view. Here is a "template-view" needed. '/users' is not optimal. This probably has to be the dashboard, if it is created via angular. At the moment the users view is the best choice.